python/composio/cli/add.py

```diff
@@ -324,22 +324,21 @@ def _handle_basic_auth(
     labels: t.Optional[t.List] = None,
 ) -> None:
     """Handle basic auth."""
```

Comment:
Good improvement! This change eliminates duplicate user prompts and makes the authentication flow more user-friendly. Consider enhancing the docstring to better describe the function's purpose and parameters.